taccsite_cms/static/site_cms/css/src/_themes/export.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Export appropriate theme file based on settings value
+ */
+const merge = require('merge-lite');
+const rootPath = __dirname + '/../../../../../../';
+const settings = require( rootPath + 'taccsite_cms/settings.json');
```

Comment:
If we need to reference this path anywhere else in the CMS it might be worth moving this path into `settings`. Then we could set the path like like `__dirname + settings_path`. An alternative approach would be to create an alias for the path if that's an option.

This is just a suggestion though.

Reply:
**Thank you.** I have referenced this suggestion at [PR #192](https://github.com/TACC/Core-CMS/pull/192#discussion_r649229084) (GH-101/#222 is piggy-backing off that isolated but un-merged feature). Your comment has made me look harder at the feature. It can be improved. If you are willing to assist further, please see [the current status of the feature](https://github.com/TACC/Core-CMS/pull/192#issuecomment-858704155).